react-native-template-ptl/template.config.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
Let's change all license headers to match the new license header for Meta Platforms, Inc.

```suggestion
 * Copyright (c) Meta Platforms, Inc. and affiliates.
```